supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[blueprint]]
+==== image:images/yes.png[yes] blueprint (noun)
+*Description*: Blueprints are simple text files in Tom's Obvious, Minimal Language (TOML) format that describe which packages, and what versions, to install into the image. They can also define a limited set of customizations to make to the final image.
+
+*Use it*: yes
+
+*Incorrect forms*: blue print, BluePrint
+
+*See also*:
+
+[discrete]
+[[certificate]]
+==== image:images/yes.png[yes] certificate (noun)
+*Description*: A certificate is an electronic document used to identify an individual, a server, a company, or other entity and to associate that identity with a public key. Such as a driver's license or passport, a certificate provides generally recognized proof of a person's identity. Public-key cryptography uses certificates to address the problem of impersonation.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate-authorities[certificate-authorities]
+
+[discrete]
+[[certificate-authorities]]
+==== image:images/yes.png[yes] Certificate Authorities (noun)
+*Description*: An entity that issues digital certificates. In {RH} {IPA}, the primary CA is `ipa`, the IdM CA. The `ipa` CA certificate is one of the following types:
+--
+* Self-signed. In this case, the `ipa` CA is the root CA.
+* Externally signed. In this case, the `ipa` CA is subordinated to the external CA.
+--
+In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certificates are one of the following types:
+
+* Signed by the `ipa` CA.
+* Signed by any of the intermediate CAs between itself and `ipa` CA. The certificate of a sub-CA cannot be self-signed.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate[certificate]
+
+[discrete]
+[[commit]]
+==== image:images/yes.png[yes] commit (noun)
+*Description*: A release or image version of the operating system. Image Builder generates an ostree commit for RHEL for Edge images. You can use these images to install or update RHEL on Edge servers.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[compose]]
+==== image:images/yes.png[yes] compose (noun)
+*Description*: Composes are individual builds of a system image, based on a particular version of a particular blueprint. Compose as a term refers to the system image, the logs from its creation, inputs, metadata, and the process itself.
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:blueprint[blueprint]
+
+[discrete]
+[[control-node]]
+==== image:images/yes.png[yes] control node (noun)
+*Description*: ny machine with Ansible installed. You can run commands and playbooks, invoking /usr/bin/ansible or /usr/bin/ansible-playbook, from any control node. You can use any computer that has Python installed on it as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+*Use it*: yes
```

Comment:
This use it line is being added to the end of the above paragraph. Can you add a blank line ahead of it so that it renders on its own line?

Same for the above entry as well.